erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-201-report.md

```diff
@@ -0,0 +1,20 @@
+# UAE VAT 201 Report in ERPNext
+
+To generate UAE VAT 201 Report in ERPNext go to <br>
+**Accounting > Value-Added Tax (VAT UAE) > UAE VAT 201** <br>
+or simply search for UAE VAT 201 in awesomebar.
+
+<img class="screenshot" alt="UAE VAT 201 Report" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-21-report.png">
+To print the report, go to Menu > Print as shown in the screenshot below.
+
+<img class="screenshot" alt="UAE VAT 201 Report Download" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-21-download.png">
+
+Note: To make sure the calculations in the report are correct, check the following:
+
+1. Correct Emirate is selected for each Address.
+
+2. Zero Rated or Exempted Items have "Is Zero Rated" or "Is Exempt" checkbox checked in the item master.
+
+3. ALL VAT accounts are selected in UAE VAT Settings.
+
+<img class="screenshot" alt="UAE VAT Account Settings" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-account-settings.png">
```

Comment:
change this screenshot... crop it from bottom

Reply:
done